### DISCUSSION TITLE
OKD 4.7: Nodes not boot: clocksource: Switched to clocksource tsc

### DISCUSSION THREAD

**Discussion by ToibayevNariman** [okd-project/okd#734] [Q&A]
**Describe the bug**

Cluster: 3-master, 3 worker nodes
Version: 4.7.0-0.okd-2021-03-07-090821
Method: UPI
Platform: vSphere 
VM created from vSphere template: fedora-coreos-33.20210217.ova

At some point, two cluster nodes (1-master,1-worker) became unavailable. The SSH connection attempt was unsuccessful. In vSphere I see the OS is not boot. OS does not receive an IP address from a DHCP server
![image](https://user-images.githubusercontent.com/2827188/114828222-77ef9f00-9deb-11eb-8e8b-bda812296830.png)

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)
